app/views/users/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+ <%= form_for(@user, as: :user, url: user_path, authenticity_token: form_authenticity_token)do |user|%>
+   <%= render 'shared/error_messages' %>
+  <%= user.label(:username, 'Username') %>
+  <%= user.text_field(:username)%>
+  <%= user.label(:email, 'Email address' ) %>
+   <%= user.text_field(:email)%>
+  <%= user.label(:password, 'Password' ) %>
+   <%= user.password_field(:password)%>
+  <%= user.submit( 'Submit' ) %>
+<% end %>  
```

Comment:
By default rails doens't populate `password_field` value, that's why the password value won't show on the edit form. We can show it by using `<%= user.password_field(:password, value: @user.password)%>`